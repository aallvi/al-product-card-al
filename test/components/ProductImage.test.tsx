// import * as React from 'react';
import React from "react";
import renderer from 'react-test-renderer'
import ProductCard, { ProductImage,  } from '../../src/components'
import {  product2 } from "../data/products";




describe('ProductTitle', () => { 


  
     test('should show component with image perzonalied', () => { 

        const wrapper = renderer.create(
                 
                            <ProductImage img="http://hola.jpg" />
               
        )

        expect(wrapper.toJSON()).toMatchSnapshot()




      })
     test('should show component image', () => { 

        const wrapper = renderer.create(
                    <ProductCard  product={product2} >

                {     () => (
                            <ProductImage/>
                        )
                    }

                    </ProductCard>
        )

        expect(wrapper.toJSON()).toMatchSnapshot()
     



      })



 })