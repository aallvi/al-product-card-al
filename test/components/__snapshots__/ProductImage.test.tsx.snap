// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle should show component image 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product image"
    className=" undefined productImg"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductTitle should show component with image perzonalied 1`] = `
<img
  alt="Product image"
  className=" undefined productImg"
  src="http://hola.jpg"
/>
`;
